equinix/resource_fabric_service_profile.go

```diff
@@ -1133,40 +1128,40 @@ func marketingInfoToFabric(schemaMarketingInfo []interface{}) v4.MarketingInfo {
 	return marketingInfos
 }
 
-func processStepToFabric(processStepl []interface{}) []v4.ProcessStep {
-	if processStepl == nil {
+func processStepToFabric(processSteps []interface{}) []v4.ProcessStep {
+	if processSteps == nil {
 		return []v4.ProcessStep{}
 	}
-	var processSteps []v4.ProcessStep
-	for _, processStep := range processStepl {
-		psTitle := processStep.(map[string]interface{})["title"].(interface{}).(string)
-		psSubTitle := processStep.(map[string]interface{})["sub_title"].(interface{}).(string)
-		psDescription := processStep.(map[string]interface{})["description"].(interface{}).(string)
-		processSteps = append(processSteps, v4.ProcessStep{
+	var processStepRes []v4.ProcessStep
```

Comment:
We should initiate this with make so that we can make it the same length as the input. Then we should also name it `processStepsRes` to account for the plural.